### PR TITLE
Add stake parameters to CommonParams

### PR DESCRIPTION
**Pull request by sgkim126**

### CODE REVIEW

**Comment by sgkim126** [comment]
It also contains the change of https://github.com/CodeChain-io/codechain/pull/1597 I'll remove do-not-merge after merging #1597.

File: types/src/common_params.rs [hunk 1]
```diff
@@ -239,10 +281,30 @@ impl Decodable for CommonParams {
         let min_asset_unwrap_ccc_cost = rlp.val_at(20)?;
         let max_body_size = rlp.val_at(21)?;
         let snapshot_period = rlp.val_at(22)?;
-        let term_seconds = if size >= 24 {
-            rlp.val_at(23)?
+
+
```
Comment (foriequal0) [inline types/src/common_params.rs]: spacing?

File: types/src/common_params.rs [hunk 2]
```diff
@@ -316,20 +385,22 @@ mod tests {
     #[test]
     fn rlp_with_term_seconds() {
```
Comment (foriequal0) [inline types/src/common_params.rs]: Should we change test name to `rlp_with_extended_fields`?

Comment (sgkim126) [reply types/src/common_params.rs]: Looks good. I'll fix it.

File: types/src/common_params.rs (hunk 1 above)
Comment (sgkim126) [reply types/src/common_params.rs]: I'll remove one line.

**Comment by sgkim126** [comment]
#1597 is merged

**Pull request merged**